README.md

```diff
@@ -1,8 +1,6 @@
 # Logging Zeek Output to Kafka
```

Comment:
@ottobackwards what do you think about adding a reference to Apache Metron/`metron-bro-plugin-kafka` once the December 2020 minutes are uploaded to https://www.apache.org/foundation/records/minutes/2020/?  Open to where to put it - top, middle, bottom...

Reply:
I think we we do that as a follow on.